.gitignore

```diff
@@ -47,3 +47,6 @@ win32/common
 **/runner.log
 **/runner.trs
 **/test-suite.log
+/.vs

```

Comment:
I believe you can put these in a separate `.gitignore` file.  I don't mind the `.vs` entry, but the other two are getting ridiculously specific.

Reply:
Oops, sorry. In VS these folders have been displayed as changes and I just added them to .gitignore without thinking about the fact that everyone would see this. Kinda new to git, in our company we have to use TFS :)
I reverted the .gitignore file and using .git/info/exclude now.